common/user/index.ts

```diff
@@ -294,6 +294,18 @@ export async function getUsers(userIds: User['userID'][]): Promise<User[]> {
     return [...students, ...pupils];
 }
 
+export async function updateLastLogin(user: User) {
```

Comment:
What about the metnor?

Reply:
I don't think Mentors are a thing anymore. I did a quick search and only was able to find them in the dev.ts setup, but there don't seem to be any actual usages. The `userType` doesn't specify mentors: https://github.com/corona-school/backend/blob/894a832bba8a1db12cc1ecbb4104e26dfa6e43b6/common/user/index.ts#L157

Reply:
I guess we should create an issue to remove it fully from the backend